feat: add IDX support to other samples (#1746)
firebase/genkit: samples/README.md

- [js-angular](js-angular/): Demo of streaming to an Angular frontend
- [js-schoolAgent](js-schoolAgent/): A simple school assistant system with a routing agent and specialized agents
- [prompts](prompts/): Shows off several prompting techniques

> These are also avaiable in IDX, Google's Cloud-Basd IDE try them out there.

Reviewer: ```suggestion
These are also available in IDX, Google's Cloud-Based IDE, for you to try.
```
